bin/buildDatabase.js

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env node
 
 import buildDatabase from '../lib/buildDatabase.js';
+import program from "commander";
 
-buildDatabase();
```

Comment:
Like we talked about today, this probably isn't working right now. I/we can update the `buildDatabase.js` script to account for the latest changes to the build process tomorrow.